packages/venia-concept/src/util/makeUrl.js

```diff
@@ -40,35 +39,38 @@ const resizeBase = joinUrls(
  * @param {number} props.width - the desired resize width of the image
  */
 const makeOptimizedUrl = (path, { type, width } = {}) => {
-    const { location } = window;
-    const urlObject = new URL(path, location.href);
-    const params = new URLSearchParams(urlObject.search);
+    const isAbsolute = absoluteUrlRE.test(path);
```

Comment:
Starting in `2.3.1`, GraphQL started returning all images as absolute URLs, which is why we had to do away with this logic. This results in only the `width` parameter being appended to these URLs, but it does not add the other two for `webp` or `pjpg` fallback. You will need to come up with another detection method other than absolute URL.

Reply:
Thanks for the tip. We now have a boolean env var for determining whether to use the backend URL or relative URLs to do image optimization. This replaces `USE_FASTLY` and I think it's clearer. What do you think?

Reply:
I really like this change, great improvement.